packages/firestore/src/local/persistence_transaction.ts

```diff
@@ -16,6 +16,9 @@
  */
 
 import { ListenSequenceNumber } from '../core/types';
+import { SimpleDb, SimpleDbStore } from './simple_db';
+import { fail } from '../util/assert';
+import { IndexedDbTransaction } from './indexeddb_transaction';
```

Comment:
This dependency seems backwards cyclically.

Reply:
Removed in later commit.